discord-bot/bot.js

```diff
@@ -0,0 +1,123 @@
+require('dotenv').config()
+let XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest
+const Discord = require('discord.js')
+const bot = new Discord.Client()
+const token = process.env.TOKEN_OSDC
+const Http = new XMLHttpRequest();
+
+const regex = 'https://imgs.xkcd.com/comics'
+const png = 'png'
+
+bot.on('ready', () => {
+    console.log('The bot is online!!!!')
+})
+
+
+//Greeting message for a New user!!!
+
+bot.on('guildMemberAdd',member => {
+    const channel = member.guild.channels.cache.find(ch => ch.name === 'member-log');
+    if(!channel) return ;
+    message.channel.send(`Hello ${message.author} Welcome to OSDC discord Server..Please introduce yourself!!`)
+})
```

Comment:
I think discord already has some cool welcome messages. We won't be needing them here ig.

What do you think @osdc/2020-21 ?

Reply:
I guess we would just need the introduction part though. What do y'all think?